ui/imports/shared/stores/send/TransactionStore.qml

```diff
@@ -108,15 +110,15 @@ QtObject {
     }
 
     function getAsset(assetsList, symbol) {
-        for(var i=0; i< assetsList.count;i++) {
-            if(symbol === assetsList.rowData(i, "symbol")) {
+        for(var i=0; i< assetsList.rowCount();i++) {
```

Comment:
Instead of this `for` loop, what about:
```cpp
Q_INVOKABLE QVariantList getAll(QAbstractItemModel* model,
                                    const QString& roleName,
                                    const QString& filterRoleName,
                                    const QVariant& filterValue) const;
``` 
(from C++)

I probably forgot to expose it in ModelUtils.qml; can you pls add it and use it here? :)

Reply:
Hey I dont think that will help here as I need multiple roles not just one